### DISCUSSION TITLE
Clarification on PR Submission for Enhancement of Sunbird inQuiry

### DISCUSSION THREAD

**Discussion by rajnishdargan** [Sunbird-inQuiry/Community#112] [Q&A]
Hello Sunbird inQuiry team,

I hope this message finds you well.

I am currently working on an enhancement of Sunbird inQuiry to bring in new types of questions 
([Match The Following](https://github.com/orgs/Sunbird-inQuiry/discussions/110) and [Arrange Sequence Question](https://github.com/orgs/Sunbird-inQuiry/discussions/111)), specifically on top of the release-8.0.0 branch for both the [editor](https://github.com/Sunbird-inQuiry/editor/tree/release-8.0.0) and [player](https://github.com/Sunbird-inQuiry/player/tree/release-8.0.0).
Our goal is to contribute these enhancements back to the community.

Given the recent updates regarding the project's support and potential transition, I would like to inquire about a few details to ensure our contributions are aligned with the upcoming plans:

1. Could you please provide any tentative date for the official release of version [8.0.0](https://inquiry.sunbird.org/use/release-notes/inquiry-release-v8.0.0-ongoing) of Sunbird inQuiry?
2. When is the next release cycle expected to start, and which branch would be appropriate for us to target for our Pull Request?

We want to make sure that our contributions are timely and aligned with the community's needs.

Thank you for your guidance and support.
cc @vinukumar-vs @mohitga @krgauraw @vrayulu @pallakartheekreddy  @ashokreddy1208 @vaivk369

**Comment by rajnishdargan** [comment]
Hi team,

Following up on my earlier message regarding our enhancements for new question types (MTF and Arrange Sequence) on the release-8.0.0 branch. Could you please update on:

1. Tentative release date for version 8.0.0.
2. Appropriate branch for our PR and next release cycle start.

Thank you!

cc: @vinukumar-vs @mohitga @krgauraw @vrayulu @pallakartheekreddy @ashokreddy1208 @vaivk369 @parthlawate @coolbung

**Comment by mohitga** [comment]
Hi @rajnishdargan, 

Sunbird is undergoing restructuring, It will take sometime before sunbird 8.0.0 is released. 
You can merge share you PR to 8.0.0 release branch only.
